### PR TITLE
Fix flaky upload form e2e test

### PR DESCRIPTION
**Pull request by dianabarsan**
# Description

https://github.com/medic/angular10-migration/issues/144

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by dianabarsan** [comment]
Please review, @njogz . Thanks!

File: tests/utils.js [hunk 1]
```diff
@@ -707,9 +707,9 @@ module.exports = {
   closeReloadModal: closeReloadModal,
 
   closeTour: async () => {
-    await element.all(by.css('.modal-dialog a.cancel')).each(async elm => {
-      await elm.click();
-    });
+    const closeButton = element(by.css('#tour-select a.btn.cancel'));
+    await browser.wait(() => closeButton.isPresent(), 5000);
```
Comment (njogz) [inline tests/utils.js:711]: Why did you decide not to use the helper functions e.g. `waitElementToBeClickable` or `waitElementToPresentNative`?

**Review by njogz — COMMENTED** [review]
Left one question.

File: tests/utils.js (hunk 1 above)
Comment (dianabarsan) [reply tests/utils.js:711]: I think we'd have a circular dependency. The utils file doesn't depend on the helper file :)

Comment (njogz) [reply tests/utils.js:711]: You're right. Totally missed that :)

**Review by njogz — APPROVED** [review]

**Pull request merged**